src/test/routes/receiver.ts

```diff
@@ -60,6 +60,7 @@ describe('Login receiver', async () => {
           claimStoreServiceMock.resolveRetrieveByClaimantIdToEmptyList()
           claimStoreServiceMock.resolveRetrieveByDefendantIdToEmptyList()
           draftStoreServiceMock.resolveFindNoDraftFound()
+          draftStoreServiceMock.resolveFindNoDraftFound()
```

Comment:
When we use `draftStoreServiceMock.resolveFindNoDraftFound` we should pass what draft type has not been found otherwise it is a bit magic.